test/parallel/test-cli-help.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+
+// test --help
+var child = spawn(process.execPath, ['--help']);
+child.stdout.on('data', function(chunk) {
+  assert.equal(/Usage/.test(chunk.toString()), true);
```

Comment:
This is not 100% sound, the output can get split over multiple chunks.  Can I suggest you use spawnSync instead?